libs/shared/src/consts/providers/channels/email.ts

```diff
@@ -62,12 +62,12 @@ export const emailProviders: IProviderConfig[] = [
     logoFileName: { light: 'sendgrid.png', dark: 'sendgrid.png' },
   },
   {
-    id: EmailProviderIdEnum.Sendinblue,
-    displayName: 'Sendinblue',
+    id: EmailProviderIdEnum.Brevo,
```

Comment:
Unfortuently this change is not backwards compatible for scheduled jobs using `sendinblue` and they will just fail. My suggestion is to create backwards compatible approach where both enums still exist, and during the handler selection we process both sendinblue and brevo using the same handler.

Reply:
Your feedback has been implemented @scopsy, thanks for the review.